static/refcache.json

```
@@ -691,6 +695,10 @@
     "StatusCode": 200,
     "LastSeen": "2025-02-02T17:58:04.819Z"
   },
+  "https://crates.io/crates/opentelemetry-instrumentation-actix-web": {
+    "StatusCode": 404,
```

Creates.io always returns 404 when we do link checking. See:

- https://github.com/rust-lang/crates.io/issues/788
- https://github.com/open-telemetry/opentelemetry.io/issues/6175

The link is valid, so let's change that:

```suggestion
    "StatusCode": 206,
```